packages/purpose-process/test/deletePurposeVersion.test.ts

```diff
@@ -90,6 +109,195 @@ describe("deletePurposeVersion", () => {
 
     vi.useRealTimers();
   });
+  it("should succeed when requester is Consumer Delegate and the Purpose is in a deletable state", async () => {
```

Comment:
```suggestion
  it.each(deletableStates)("should succeed when requester is Consumer Delegate and the Purpose is in %s state", async (state) => {
```
use an `it.each` in order to iterate all deletable states, declare the `const deletableStates` at the beginning